dash/development/_r_components_generation.py

```diff
@@ -369,6 +399,18 @@ def write_help_file(name, props, description, prefix, rpkg_data):
         for p in prop_keys
     )
 
+    if re.findall(r"(?<!\\)%", item_text):
+        print(
+            "Error, aborting R package generation: "
+            "percent signs cannot be used in docstrings "
+            "without escaping with a backslash, e.g; \\%.\n"
+            "Please omit or escape any percent signs in "
+            "your JavaScript source before rebuilding the "
+            "R package.",
+            file=sys.stderr,
+        )
+        sys.exit(1)
```

Comment:
As discussed on slack: javascript docstrings should NOT contain escapes for % characters, that would look weird in JS (and in Python). So the right way to handle this is to automatically insert the escapes when translating from JS to R.

Reply:
fixed in 52a3394ad4f07b9bd92d1eec9e4ff4ca29b4ef63